ansible/roles/controller/tasks/deploy.yml

```diff
@@ -236,6 +236,14 @@
 
       "CONFIG_whisk_transactions_header": "{{ transactions.header }}"
 
+- name: merge extra env variables
+  set_fact:
+    controller_env: "{{ controller_env | default({}) | combine(item) }}"
+  with_items: "{{ extraEnvSets.split(',') | map('trim') | map('extract', vars)| list }}"
+  loop_control:
+    label: redacted
```

Comment:
Ansible would by default emit the item value to stdout. As the values may contain secrets set the label to a constant